app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,42 @@
+class CommentsController < ApplicationController
+  # customary in rails to put actions in order like:
+  # index, show, new, edit, create, update, destroy
+  def index
+
+  end
+
```

Comment:
If you are not using these methods or associated views you can actually just take them out